tests/load-tests.php

```diff
@@ -97,6 +97,32 @@ function( $default, $option, $passed_default ) use ( &$has_passed_default ) {
 		$this->assertSame( $new_value, $settings );
 	}
 
+	/**
+	 * @dataProvider data_legacy_modules
```

Comment:
@mukeshpanchal27 Missing `@test` here.

Reply:
@mehulkaklotar To run test we can use two methods.
- At starting of function name use `test_`
- Or you can use the `@test` annotation in a method’s docblock.